scripts/hardhat-node

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+#start hardhat-node in a way that "run-with-testrpc" will work
+# (it looks up for "Listening on" string, to determine when the node is up)
+trap "pkill -9 -f 'hardhat node'" EXIT
```

Comment:
how does it work? is it needed on local environment?

Reply:
trap command gets a bash command and signal to work on, so whenever this script exists, it kills "hardhat node" that the script itself ran, so we don't have a running node in the background once we finish.

Reply:
nice